### PR TITLE
Add diagram GRC 2.3

### PR DESCRIPTION
**Pull request by dockerymick**
Adding diagram for governance arch

https://github.com/open-cluster-management/backlog/issues/14196

### CODE REVIEW

File: risk_compliance/grc_intro.adoc [hunk 1]
```diff
@@ -8,7 +8,7 @@ Enterprises must meet internal standards for software engineering, secure engine
 
 Enhance the security for your cluster with the {product-title} governance lifecycle. The product governance lifecycle is based on defined policies, processes, and procedures to manage security and compliance from a central interface page. View the following diagram of the governance architecture:
 
-image:../images/security-arch.png[Governance architecture diagram]
+image:../images/governance_arch_2.3.png[Governance architecture diagram]
```
Comment (dockerymick) [inline risk_compliance/grc_intro.adoc:11]: @gparvin since there are some new components, are there any that need to be described?

Comment (gparvin) [reply risk_compliance/grc_intro.adoc:11]: Good point.  The new content is here: https://github.com/open-cluster-management/rhacm-docs/blob/428a1fc35c2d4202356d47217ae70a6d599e5d00/risk_compliance/ansible_grc.adoc#configuring-ansible-tower-for-governance
Can we add a link to that page at the end of the bullet **Policy-based governance framework**.   Something like:
When policies are violated, [automation can be configured](https://github.com/open-cluster-management/rhacm-docs/blob/428a1fc35c2d4202356d47217ae70a6d599e5d00/risk_compliance/ansible_grc.adoc#configuring-ansible-tower-for-governance) to run to take any actions the user chooses.

**Review by gparvin — COMMENTED** [review]
See the comment that suggests adding a link to our ansible configuration page.

File: risk_compliance/grc_intro.adoc (hunk 1 above)
Comment (dockerymick) [reply risk_compliance/grc_intro.adoc:11]: Sure !

**Comment by dockerymick** [comment]
Based on the review and approval via Slack from @gparvin, I am merging this PR

**Pull request merged**